akka-remote/src/main/resources/reference.conf

```diff
@@ -134,7 +134,37 @@ akka {
         # Level 1 strongly prefer low CPU consumption over low latency.
         # Level 10 strongly prefer low latency over low CPU consumption. 
         idle-cpu-level = 5
+
+        # compression of common strings in remoting messages, like actor destinations, serializers etc
+        compression {
+          # global flag to disable all compression
+          enabled = off
+          
+          # unlocks additional very verbose debug logging of compression events (on DEBUG log level)
+          debug = off
+          
+          actor-refs {
+            enabled = off
+            # Max number of compressed actor-refs
+            # Note that compression tables are "rolling" (i.e. a new table replaces the old 
+            # compression table once in a while), and this setting is only about the total number
+            # of compressions within a single such table.
+            # Must be a positive natural number.
+            max = 256
+          }
+          manifests {
+            # enables
+            enabled = off
```

Comment:
when we have tried out things I think we can remove these `enabled` flags, I see no reason why one would not want to use compression